misp.py

```diff
@@ -961,9 +1015,30 @@ def to_ids(self, value):
         self._to_ids = int(value)
 
     @property
+    def server(self):
+        return self._server
+
+    @to_ids.setter
+    def server(self, value):
+        self._server = value
+
+    @property
     def ShadowAttribute(self):
         return None
 
+    def download(self):
```

Comment:
It made more sense to me to have download attached to the attribute because it is what you want to download. But by reconsidering this right now, indeed it would avoid this ugly _server attribute